docs/operations/mixed-workloads.md

```diff
@@ -0,0 +1,211 @@
+You can use the built-in [“high/low” laning strategy](../configuration/index.md#highlow-laning-strategy), or [define your own laning strategy manually](../configuration/index.md#manual-laning-strategy).
+* `druid.query.scheduler.numThreads` – The total number of queries that can be served per Broker. We recommend setting this value to 1-2 less than `druid.server.http.numThreads`.
+  > The query scheduler by default does not limit the number of Broker HTTP threads. Setting this property to a bounded number limits the thread count. If the allocated threads are all occupied, any incoming query, including interactive queries, will be rejected with an HTTP 429 status code.
+
+### Lane-specific properties
+
+If you use the __high/low laning strategy__, set the following:
+
+* `druid.query.scheduler.laning.maxLowPercent` – The maximum percent of query threads to handle low priority queries. The remaining query threads are dedicated to high priority queries.
+
+Consider also defining a [prioritization strategy](../configuration/index.md#prioritization-strategies) for the Broker to label queries as high or low priority. Otherwise, manually set the priority for incoming queries on the [query context](../querying/query-context.md).
+
+If you use a __manual laning strategy__, set the following:
+
+* `druid.query.scheduler.laning.lanes.{name}` – The limit for how many queries can run in the `name` lane. Define as many named lanes as needed.
+* `druid.query.scheduler.laning.isLimitPercent` – Whether to treat the lane limit as an exact number or a percent of the minimum of `druid.server.http.numThreads` or `druid.query.scheduler.numThreads`.
+
+With manual laning, incoming queries can be labeled with the desired lane in the `lane` parameter of the [query context](../querying/query-context.md).
+
+See [Query prioritization and laning](../configuration/index.md#query-prioritization-and-laning) for additional details on query laning configuration.
+
+### Example
+
+Example config for query laning with the high/low laning strategy:
+
+```
+# Laning strategy
+druid.query.scheduler.laning.strategy=hilo
+druid.query.scheduler.laning.maxLowPercent=20
+
+# Limit the number of HTTP threads for query processing
+# This value should be less than druid.server.http.numThreads
+druid.query.scheduler.numThreads=40
+```
+
+
+## Service tiering
+
+In service tiering, you define separate groups of Historicals and Brokers to manage queries based on the segments and resource requirements of the query.
+You can limit the resources that are set aside for certain types of queries.
+Many heavy queries involving complex subqueries or large result sets can crash a Broker, or worse, take down a cluster.
+Minimize the impact of these heavy queries by limiting them to a separate Broker tier.
+When all Brokers set aside for heavy queries are occupied, users must wait to submit additional heavy queries until the designated resources become available.
```

Comment:
This looks a bit unclear to me. I think it's because there is nothing in Druid disallowing users to submit additional queries. Perhaps 

```suggestion
When all Brokers set aside for heavy queries are occupied, additional queries submitted subsequently can wait until the designated resources become available, which can fail those queries after query timeout expires.
```

Reply:
```suggestion
When all Brokers set aside for heavy queries are occupied, subsequent queries must wait until the designated resources become available.
A prolonged wait results in the later queries failing with a timeout error.
```